kgitb/config.py

```diff
@@ -1,8 +1,9 @@
+# -*- coding: utf-8 -*-
 from os import environ as env
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a docstring to this module. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)